lib/datadog/tracing/trace_operation.rb

```diff
@@ -373,6 +373,11 @@ def fork_clone
         )
       end
 
+      def self.appsec_standalone_reject?(trace)
+        Datadog.configuration.appsec.standalone.enabled &&
+          (trace.nil? || trace.get_tag(Datadog::AppSec::Ext::TAG_DISTRIBUTED_APPSEC_EVENT) != '1')
+      end
+
```

Comment:
I suggest moving this method to the `AppSec` namespace, since it doesn't really belong to the trace operation, and is a "static" method.

Reply:
Fixed in e24310ef